Formula/metabase.rb

```diff
@@ -21,6 +21,8 @@ class Metabase < Formula
   def install
     if build.head?
       Language::Node.setup_npm_environment
+      # help yarn find npm's node-gyp binary inside superenv
+      ENV.prepend_path "PATH", "#{Formula["node"].opt_libexec}/lib/node_modules/npm/bin/node-gyp-bin"
```

Comment:
Ideally we should either fix this inside the yarn formula somehow or we should create a language/node helper for it too, so that it's $PATH is only managed in one place for the exact same reason we use `setup_npm_environment` now.

And just the clarify: homebrew's `yarn` is able to compile native addons with `node-gyp` outside a formula completely fine. It's superenv which break yarn ability to find `node-gyp` when used in formulas a a build dependency.